### PR TITLE
html-forms - Raene Horist

### PR DESCRIPTION
**Pull request by Raenbow**
https://user-images.githubusercontent.com/34592143/164317041-710a6725-ca00-4f0f-8a8d-142975f21744.mp4

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        </div>
+      </div>
+      <div>
+        <label>Comments</label>
+        <div>
+          <textarea name="comments" cols="25" rows="10"></textarea>
+        </div>
+      </div>
+      <div>
+        <button>Submit</button>
```
Comment (TimelikeClosure) [inline html-forms/index.html]: It's generally a good idea to specify the `type` attributes on `<button>`s, even if they have the default `type`.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  </head>
+  <body>
+    <h1>
+      Coding Form!
+    </h1>
+    <form action="">
+      <div>
+        <label for="email">Email</label>
+        <div>
+          <input type="text" name="email" />
```
Comment (TimelikeClosure) [inline html-forms/index.html]: For `<input>`s that should hold email addresses, try `type="email"`.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by Raenbow** [comment]
@TimelikeClosure I made the changes you suggested as well as adding labels to the radio inputs and adjusting the 'for' attributes on the labels.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    </h1>
+    <form action="">
+      <div>
+        <label for="email">Email</label>
+        <div>
+          <input type="email" name="email" />
+        </div>
+      </div>
+      <div>
+        <label for="reasons">Why is coding cool?</label>
```
Comment (TimelikeClosure) [inline html-forms/index.html]: As counter-intuitive as it is, a `<label>`'s `for` attribute refers to a form control by `id`, not by `name`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**